src/OrchardCore/OrchardCore.Search.Elasticsearch.Core/Services/ElasticIndexInitializerService.cs

```diff
@@ -13,12 +13,12 @@ namespace OrchardCore.Search.Elasticsearch;
 /// Provides a way to initialize Elasticsearch index on startup of the application
 /// if the index is not found on the Elasticsearch server.
 /// </summary>
-public class ElasticIndexInitializerService : ModularTenantEvents
+public sealed class ElasticIndexInitializerService : ModularTenantEvents
 {
     private readonly ShellSettings _shellSettings;
     private readonly ElasticIndexManager _elasticIndexManager;
     private readonly ElasticIndexSettingsService _elasticIndexSettingsService;
-    protected readonly IStringLocalizer S;
+    private readonly IStringLocalizer S;
```

Comment:
also move it to like 23 and add a new line before it.
```suggestion

    internal readonly IStringLocalizer S;
```